src/ConEmu/RealConsole.cpp

```diff
@@ -4846,7 +4846,7 @@ bool CRealConsole::CreateOrRunAs(CRealConsole* pRCon, RConStartArgsEx& Args,
 				hr = SHGetFolderPath(nullptr, CSIDL_SYSTEM, nullptr, SHGFP_TYPE_CURRENT, szUserDir);
 				if (FAILED(hr) || !*szUserDir)
 					wcscpy_c(szUserDir, L"C:\\");
-				lpszWorkDir = szUserDir;
+				lpszWorkDir = lstrdup(szUserDir).Detach();
```

Comment:
I think that is wrong. There is no deallocation of `lpszWorkDir` by the caller.
`lpszWorkDir` should be declared as `CEStr&` to return `szUserDir` value.
But it's simpler to just use temp var here.